src/js/containers/awardV2/AwardV2Container.jsx

```diff
@@ -122,13 +124,20 @@ export class AwardContainer extends React.Component {
     }
 
     render() {
-        return (
-            <Award
-                award={this.props.award}
-                awardId={this.props.params.awardId}
-                inFlight={this.state.inFlight}
-                noAward={this.state.noAward} />
-        );
+        const isV2url = Router.history.location.pathname.includes('award_v2');
+        let content = null;
+        if (!this.state.inFlight) {
+            if (this.props.award.category === 'idv' || isV2url) {
+                content = (<Award
+                    award={this.props.award}
+                    inFlight={this.state.inFlight}
+                    noAward={this.state.noAward} />);
+            }
+            else {
+                content = (<AwardV1Container awardId={this.props.params.awardId} />);
+            }
+        }
+        return content;
     }
```

Comment:
Do we really need `line 129` as a condition? I don't see where we're handling in the UI the case where `inFlight` is true. 

Should we be handling that here in the container, or letting the child components handle it? It appears `<Award />` wants to do something with it, but with these changes it will always be `true`.